scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/service/ScheduleManagerTest.java

```diff
@@ -138,7 +136,7 @@ private void assertCreateAndFindAllSchedules(int noOfSpecificDateSchedules) {
 		assertSpecificSchedulesFoundEquals(noOfSpecificDateSchedules, foundSpecificSchedules);
 
 		// reset all records for next test.
-		afterTest();
+		removeAllRecoredsFromDatabase();
```

Comment:
This doesn't make sense.
We shouldn't be doing clean up in the middle unless the test case requires it.
If you need the cleanup, then it should be a separate test given how this method is used.